src/tests/class_tests/openms/source/ConsensusIDAlgorithmWorst_test.cpp

```diff
@@ -107,52 +107,40 @@ START_SECTION(void apply(std::vector<PeptideIdentification>& ids))
   hits = f[0].getHits();
   TEST_EQUAL(hits.size(), 11);
 
-  TEST_EQUAL(hits[0].getRank(), 1);
   TEST_EQUAL(hits[0].getSequence(), AASequence::fromString("F"));
   TEST_REAL_SIMILAR(hits[0].getScore(), 0.0);
 
-  TEST_EQUAL(hits[1].getRank(), 2);
   TEST_EQUAL(hits[1].getSequence(), AASequence::fromString("G"));
   TEST_REAL_SIMILAR(hits[1].getScore(), 0.2);
 
-  TEST_EQUAL(hits[2].getRank(), 3);
   TEST_EQUAL(hits[2].getSequence(), AASequence::fromString("C"));
   TEST_REAL_SIMILAR(hits[2].getScore(), 0.3);
 
   // hits with the same score get assigned the same rank:
```

Comment:
```suggestion
```

Reply:
this use case is not directly supported anymore. would need to be done on the score itself